js/src/chrome/projects/chrome.projects.js

```diff
@@ -1458,6 +1463,22 @@ Wu.Chrome.Projects = Wu.Chrome.extend({
 		app.Controller.openFirstProject();
 		
 	},
+
+	_onProjectChanged : function (e) {
```

Comment:
@knutole  current architecture is like > We fire Events from Model Layer and receive the fired Event inside Chrome layer . Exm-> https://github.com/mapic/mapic.js/blob/master/js/src/models/model.project.js#L440 this fires and which is received at https://github.com/mapic/mapic.js/blob/master/js/src/chrome/projects/chrome.projects.js#L1451

And the previous ProjectChanged Event was never got fired , it was in model layer.